src/Workspaces/CSharp/Portable/Formatting/Rules/SpacingFormattingRule.cs

```diff
@@ -326,11 +326,12 @@ public override AdjustSpacesOperation GetAdjustSpacesOperation(SyntaxToken previ
             }
 
             // Index expressions
-            if (previousKind == SyntaxKind.CaretToken && previousParentKind == SyntaxKind.IndexExpression)
+            if (previousKind == SyntaxKind.CaretToken && previousParentKind == SyntaxKindEx.IndexExpression)
```

Comment:
well this it extremely strange...

Reply:
Code style layer compiles against 15.7, but supports formatting new C# 8 syntax (aside from range expressions because that would be more complex than I wanted to do here).